provision/provision-helpers.sh

```diff
@@ -731,3 +732,13 @@ function vvv_search_replace_in_file() {
   fi
 }
 export -f vvv_search_replace_in_file
+
+# @description Identify the service name for the MySQL service via /etc/init.d/ contents
```

Comment:
in the past both have worked and been alias'd somehow

Reply:
that wasn't something we did though, but an observation of how it came out the box

Reply:
exactly yea, it seems how mariadb installer works ( not vvv ). i am thinking maybe the mariadb vs mysql thing is a glitch that i experienced one time, as i can't replicate it again. my last provision, it gets correctly `mariadb`.
i will do 1 more tests, and maybe revert the changes mariadb vs mysql service thing from this PR

Reply:
so i've reverted these, it seems the issue is something in my lab machine.

i've been testing the mariadb with multiple vagrant destroy and vagrant up --provision, its consistently working fine with mariadb
```shell
vagrant@vvv:~$ sudo service mariadb status
 * /usr/bin/mariadb-admin  Ver 10.0 Distrib 10.11.9-MariaDB, for debian-linux-gnu on x86_64
Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Server version          10.11.9-MariaDB-ubu2404
Protocol version        10
Connection              Localhost via UNIX socket
UNIX socket             /run/mysqld/mysqld.sock
Uptime:                 12 sec

Threads: 1  Questions: 63  Slow queries: 0  Opens: 33  Open tables: 26  Queries per second avg: 5.250
```